scripts/regen.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+osascript -e 'quit app "Xcode"'
```

Comment:
Mind leaving a comment here explaining what this script does?

Reply:
I actually didn't mean to commit this, but it is useful when doing codegen development.  Maybe I clean it up / comment it & leave it in `scripts/` permanently...

Reply:
yeah that'd be dope

Reply:
Much more detail now